src/IFrameManager.js

```diff
@@ -233,8 +233,11 @@ class IFrameManager {
   onMessage(event) {
     const { data, origin } = event;
     // eslint-disable-next-line object-curly-newline
-    const { type, engagementId, portalId, userId, ownerId } = event.data;
+    const { type } = event.data;
     if (type === messageType.SYNC) {
+      const { type, portalId, userId, ownerId, engagementId, iframeLocation } =
```

Comment:
I'm seeing `const { type } = event.data;` outside of the `if`. Does it need to be destructured twice?

Reply:
I don't believe so